src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -357,33 +357,21 @@ private static class Logging {
      */
     private boolean streamFilterSet;
 
-    /**
-     * cache LUDCL (Latest User Defined Class Loader) till completion of
-     * read* requests
+    /* Cache LUDCL (Latest User Defined Class Loader) utill completion of read requests.
+     * If true LUDCL/forName results would be cached, true by default starting Java8.
```

Comment:
Please hyphenate "User-Defined" and correct the spelling of "until".
I think "with" and a space are missing in "starting with Java 8".
```java
    /* Cache LUDCL (Latest User-Defined Class Loader) until completion of read requests.
     * If true LUDCL/forName results would be cached, true by default starting with Java 8.
     */
```

Reply:
Updated